sematic/db/models/factories.py

```diff
@@ -57,7 +57,6 @@ def make_artifact(
 ) -> Artifact:
     """
     Create an Artifact model instance from a value and type.
-
```

Comment:
[Cosmetic] Please revert. Untouched file, and the first line of the docstring is supposed to be followed by an empty line.